src/home/diagnostictool/DiagnosticTool.js

```diff
@@ -121,6 +146,10 @@ const styles = StyleSheet.create({
     padding: 10,
     color: Theme.palette.white
   },
+  image2: {
```

Comment:
Come up with a better name for the image and image2 styles to something more descriptive like the names that were given to the new button styles